mix.exs

```diff
@@ -85,7 +84,6 @@ defmodule Nerves.MixProject do
         "mix.exs",
         "README.md",
         "scripts",
-        "priv/templates",
```

Comment:
Wait.... the whole source priv directory fiasco yak shaved all the way to removing Distillery!?!??! 🤣

Reply:
YOLO!!!!

Reply:
wow